Source/SwiftLintBuiltInRules/Rules/Style/SortedImportsRuleExamples.swift

```diff
@@ -309,5 +342,27 @@ internal struct SortedImportsRuleExamples {
             @_exported @testable import BBB
               import AAA
             """),
+        Example("""
+          public import AAA
+        @_exported @testable import BBB
+        """, configuration: groupByAttributesConfiguration, testMultiByteOffsets: false): Example("""
+            @_exported @testable import BBB
+              public import AAA
+            """),
+        Example("""
+        import AAA
+        private import BBB
+        fileprivate import CCC
+        internal import DDD
+        package import EEE
+        public import FFF
+        """, configuration: groupByAttributesConfiguration, testMultiByteOffsets: false): Example("""
+            public import FFF
+            package import EEE
+            internal import DDD
+            fileprivate import CCC
+            private import BBB
+            import AAA
+            """),
     ]
```

Comment:
On additional test I'd like to add is one that has comments attached to the access-level modifier (just a comment above the import statement) to see that the comments is correctly moved together with the whole import.